src/utils/platform.js

```diff
@@ -0,0 +1,11 @@
+const platform = (() => {
+  if ('userAgentData' in navigator && 'platform' in navigator.userAgentData && navigator.userAgentData.platform) {
+    return navigator.userAgentData.platform
```

Comment:
in secure contexts (https, localhost) and on newer chrome and android browsers we can check this instead of user agent string (not supported on Firefox or Safari, but they don't support WebUSB anyway)